src/NuGet.Clients/PackageManagement.UI/Models/SearchResultPackageMetadata.cs

```diff
@@ -4,41 +4,297 @@
+        private NuGetVersion _latestVersion;
+
+        public NuGetVersion LatestVersion
+        {
+            get
+            {
+                return _latestVersion;
+            }
+            set
+            {
+                if (!VersionEquals(_latestVersion, value))
+                {
+                    _latestVersion = value;
+                    OnPropertyChanged(nameof(LatestVersion));
+
+                    // update tool tip
+                    if (_latestVersion != null)
+                    {
+                        var versionForDisplay = new VersionForDisplay(_latestVersion, string.Empty);
+                        LatestVersionToolTip = string.Format(
+                            CultureInfo.CurrentCulture,
+                            Resources.ToolTip_LatestVersion,
+                            versionForDisplay);
+                    }
+                    else
+                    {
+                        LatestVersionToolTip = null;
+                    }
+                }
+            }
+        }
+
+        private string _latestVersionToolTip;
+
+        public string LatestVersionToolTip
+        {
+            get
+            {
+                return _latestVersionToolTip;
+            }
+            set
+            {
+                _latestVersionToolTip = value;
+                OnPropertyChanged(nameof(LatestVersionToolTip));
+            }
+        }
+
+        private bool _selected;
+
+        public bool Selected
+        {
+            get { return _selected; }
+            set
+            {
+                if (_selected != value)
+                {
+                    _selected = value;
+                    OnPropertyChanged(nameof(Selected));
+                }
+            }
+        }
+
+
+        private string _authorAndDownloadText;
+
+        public string AuthorAndDownloadText
+        {
+            get
+            {
+                return _authorAndDownloadText;
+            }
+            set
+            {
+                if (_authorAndDownloadText != value)
+                {
+                    _authorAndDownloadText = value;
+                    OnPropertyChanged(nameof(AuthorAndDownloadText));
+                }
+            }
+        }
+
+        private bool VersionEquals(NuGetVersion v1, NuGetVersion v2)
+        {
+            if (v1 == null && v2 == null)
+            {
+                return true;
+            }
+
+            if ((v1 == null && v2 != null) ||
+                (v1 != null && v2 == null))
+            {
+                return false;
+            }
+
+            return v1.Equals(v2, VersionComparison.Default);
+        }
+
+        private int? _downloadCount;
+
+        public int? DownloadCount
+        {
+            get
+            {
+                return _downloadCount;
+            }
+            set
+            {
+                _downloadCount = value;
+                UpdateAuthorAndDownloadText();
```

Comment:
why we are mixing the Author update with the Download count? I don't see any reason we should mix the logic